useScroll/ScrollButtons.js

```diff
@@ -102,7 +104,7 @@ const useScrollButtons = (props) => {
 
 	function onKeyDownButton (ev, position) {
 		const
-			{focusableScrollButtons, vertical, preventBubblingOnKeyDown} = props,
+			{vertical, preventBubblingOnKeyDown} = props,
```

Comment:
Wow, this issue is pretty difficult .
onKeyDown has closure value from the previous prop, and it seems that the closure value is not updated even if the prop is changed.
your approch is not fix the cause and just make sure to get the latest focusableScrollButtons.
Is it correct that I analyzed?
If right , can you add a more description about it?
I should discuss this with our members.

Reply:
Yes, you are correct. I didn't know any other approach to solve it